packages/terra-action-header/README.md

```diff
@@ -0,0 +1,25 @@
+# Terra Action Header
+
+
+[![NPM version](http://img.shields.io/npm/v/terra-action-header.svg)](https://www.npmjs.org/package/terra-action-header)
+[![Build Status](https://travis-ci.org/cerner/terra-core.svg?branch=master)](https://travis-ci.org/cerner/terra-core)
+
+The terra-action-header component is a header bar containing a title and optional actionable items.
```

Comment:
I feel like for the description, we can add a small subset of examples it items

    actionable items, such as links and buttons.

Reply:
Updated [here](https://github.com/cerner/terra-core/commit/5b230102a59e8b12ddab3baf4971a109535b0884).